LayerClient/LayerClient.py

```diff
@@ -319,6 +320,25 @@ def send_announcement(self, sender, recipients, message_parts,
             )
         )
 
+    def mark_delivery_receipt_message(self, message_uuid='', type='sent'):
```

Comment:
Since `message_uuid` is required, I think it would be better for it to be a normal argument rather than a keyword argument.